data/db-schema/src/main/resources/net/corda/db/schema/cluster-certificates/migration/certificates-creation-v1.0.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+    <property name="schema.name" value="CERTIFICATES"/>
+
+<changeSet author="R3.Corda" id="certificates-creation-v1.0">
+
+<sql>
+    CREATE SCHEMA IF NOT EXISTS CERTIFICATES;
+</sql>
+
+<createTable tableName="cluster_certificates" schemaName="${schema.name}">
+    <column name="tenant_id" type="VARCHAR(255)">
+        <constraints nullable="false"/>
+    </column>
+    <column name="alias" type="VARCHAR(255)">
+        <constraints nullable="false"/>
+    </column>
+    <column name="raw_certificate" type="TEXT">
+        <constraints nullable="false"/>
+    </column>
+</createTable>
+
+<addPrimaryKey tableName="cluster_certificates" columnNames="tenant_id, alias" constraintName="certificates_pk"
+               schemaName="${schema.name}"/>
+</changeSet>
+</databaseChangeLog>
```

Comment:
Looks good but file formatting is a bit mixed up (unless it's just github showing it weird).